src/MOI/indicator_constraint.jl

```diff
@@ -0,0 +1,117 @@
+
+function MOI.supports_constraint(
+    ::Optimizer, ::Type{MOI.VectorAffineFunction{Float64}}, ::Type{<:MOI.IndicatorSet{A, S}}
+) where {A, S <: Union{MOI.LessThan{Float64}, MOI.GreaterThan{Float64}, MOI.EqualTo{Float64}}}
+    return true
+end
+
+function MOI.is_valid(
+    model::Optimizer,
+    c::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}, S}
+) where {A, S <: MOI.IndicatorSet}
```

Comment:
Is this correct? What is `A` here?

Reply:
same as above, `A` is the MOI.ActivationCondition

Reply:
But `A` isn't used in the function, or in the header. You probably just mean `) where {S <: IndicatorSet}`?

Reply:
that's right, I hadn't caught this one thanks